sdk/ai/ai-projects/src/agents/utils.ts

```diff
@@ -40,17 +40,82 @@ export enum connectionToolType {
 }
 
 /**
- * Creates a tool definition for a connection tool with the given connection ids.
+ * Creates a connection tool
  *
  * @param toolType - The type of the connection tool.
  * @param connectionIds - A list of the IDs of the connections to use.
- * @returns The function tool definition.
+ * @returns An object containing the definition for the connection tool
  */
-export function fromConnectionId(toolType : connectionToolType, connectionIds: string[]) : ToolDefinitionParent {
+export function createConnectionTool(
```

Comment:
Since there are connection tools (bing, sharepoint, fabric) and other types of tools (file search, code interpreter, azure ai search) I wanted to keep the convenience api naming consistent for all as create____Tool. Will it be confusing if we name this method differently?